packages/web3-eth/package.json

```diff
@@ -29,6 +29,7 @@
         "web3-eth-ens": "1.0.0-beta.52",
         "web3-eth-iban": "1.0.0-beta.52",
         "web3-eth-personal": "1.0.0-beta.52",
+        "web3-eth-miner": "1.0.0-beta.52",
```

Comment:
I thought we said we were going to keep web3-eth-miner in its own module?

Reply:
Yes, i forgot to remove it.

Reply:
Done @joshstevens19.

Reply:
@joshstevens19 @princesinha19 Miner should be a standalone module. This means it doesn't have to be a dependency of the web3-eth module.

Reply:
Yes, right. @nivida